msbuild/Xamarin.MacDev.Tasks.Core/Tasks/ACToolTaskBase.cs

```diff
@@ -344,17 +352,24 @@ public override bool Execute ()
 					}
 
 					if (clone) {
-						var path = Path.Combine (intermediateCloneDir, vpath);
-						var dir = Path.GetDirectoryName (path);
+						var dest = Path.Combine (intermediateCloneDir, vpath);
+						var src = ImageAssets[i].GetMetadata ("FullPath");
+						var dir = Path.GetDirectoryName (dest);
 
 						Directory.CreateDirectory (dir);
```

Comment:
Minor, but if `src` does not exists then there was no need to create the directory (and get its name). The creation should only happen once `src` existence is confirmed.